UPDATE: Introduction to GitHub

a-a-ron opened the pull request:
The purpose of this PR is to see how we can increase the completion rate and improve the user experience. Below is the August data for this module:

| Module | Completion Rate | # completed / # started |
|--------|--------|--------|
| [Introduction to GitHub](https://learn.microsoft.com/en-us/training/modules/introduction-to-github/)| 38.74% | 1,177/3,038 |

**For reference:** See the [parent issue](https://github.com/githubpartners/microsoft-learn/issues/35) for context

## Module file structure
![Screen Shot 2022-09-28 at 2 02 42 PM](https://user-images.githubusercontent.com/6351798/192878192-b44c8eb7-c20e-4eb9-81c8-6ce65d7d2664.png)


## Items to review for this module

We need to conduct a performance review on each of these modules to see how we can increase the completion rates, improve the user experience, and maintain the level of quality that we set for our learners. 

### Items for consideration
- **Thoroughly review the module content for flow, narrative, and length**
  - Review use of images, tables, and other visual elements
  - Does the content read like a doc or like a conversation guide?
  - Does the estimated time to complete each unit remain accurate?
- **Review the introduction unit** 
  - Does it capture your attention?
  - Does it contain repetitive content from the course title page?
  - Do the learning objectives/scope align with the content? 
  - Does the target audience match the content?
  - Do we accurately describe the module and learning objectives? (Is it possible that learners start the course thinking it's one thing to find out it's another?) 
- **Review the module exercise**
  - Does it work?
  - Does it enforce a learning objective?
  - How is it placed in the module? Should we move it around?
  - Do we need other exercises?
- **Review the questions at the end of the module**
  - Do the questions test on content taught in the module?
  - Do the questions follow best practices for item writing?

## Estimated Work Effort
| Task | Hours | 
|--------|--------|
| Detailed module review | 18 hours | 
| Action items as identified from review | TBD |

a-a-ron committed:
Update 1-introduction.yml

camihmerhar committed:
Update 1-introduction.md

After walking through the content a couple of times and rereading the learning objectives a couple of things stood out to me. There were 6 learning objectives which seemed overwhelming and I felt we didn't set up the reader for what this module was truly about which are the basics.

camihmerhar committed:
Update 2-what-is-github.md

I removed the subunit "The GitHub Flow," ironically for me it felt like it broke the flow of the unit, especially being at the very beginning. If it is in fact something someone must learn before they learn about the basic features of GitHub I feel like it should be a prerequisite to the course or be its own module. I also made some minor grammatical edits.

camihmerhar committed:
Update 5-summary.md

I removed the sentence in the first paragraph about creating a GitHub page because after walking through the exercise it doesn't include it anymore and I updated the learning outcomes to be consistent with the changes I made to the launch page.

camihmerhar commented:
### Action Items For Each Module
- **Thoroughly review the module content for flow, narrative, and length**
  - Review use of images, tables, and other visual elements
     -  **_I feel like the images are great and relevant to the subunits._** 
  - Does the content read like a doc or like a conversation guide?
     **_- I feel like it reads like a conversation guide. The content is short and to the point and high level enough for people to understand_**
- **Review the introduction unit** 
  - Does it capture your attention? 
    **_- I think the content set the reader up for what it intended to do which is introduce someone to GitHub and its most common and utilized features. However the learning objectives seemed off so I had rewrote them to be more clear for the reader to anticipate what was coming. I also removed the GitHub flow subunit because it felt out of place. If we believe it fits then I would recommend making it a prerequisite or making it a new unit._** 
  - Do the learning objectives/scope align with the content? 
    **_- To me it did not, I can see how, but you really need to stop, digest the information and correlate what you did/read to the objectives_**
  - Does the target audience match the content?
    **_- In my opinion, yes and no. Yes that the content is easy to read and to the point. The only thing that seems out of place is GitHub flow subunit so I removed it._** 
  - Do we accurately describe the module and learning objectives? (Is it possible that learners start the course thinking it's one thing to find out it's another?) 
     - **_I believe learners to go into this expecting a barebones simple intro, but the learning objectives seemed confusing_**
- **Review the module exercise**
  - Does it work?
    **_- Yes!_**
  - Does it enforce a learning objective?
    **_- With the new learning objectives I believe it does_**
  - How is it placed in the module? Should we move it around?
    **_- I believe it is in a good position and it follows similar patterns of other_**  
  - Do we need other exercises?
    **_- I don't think so_**
- **Review the questions at the end of the module**
  - Do the questions test on content taught in the module?
     **_- Yes!_**
  - Do the questions follow best practices for item writing?
     **_- I believe so_**

This module is estimated at 1 hour and 12 minutes with 58 minutes dedicated to the exercise. I just did the exercise and I would estimate that it took me 30 minutes, with the shortened intro that should take a minute to read (aka removing the learning objectives) we can bring that down to a minute, the unit would take 8 minutes to read, and then the summary would be a minute. With that I believe the new total time could be 40 minutes.

camihmerhar committed:
Update index.yml

Updated the learning objectives to reflect the updated ones in the introduction unit.
